dxr/plugins/python/tests/test_modules_packages/test_modules_packages.py

```diff
@@ -0,0 +1,49 @@
+from dxr.testing import DxrInstanceTestCase
+
```

Comment:
If I search for `module:snook.eee`, will it find both `some/snook/eee/__init__.py` and `other/snook/eee/__init__.py`? If so, could we add a test for that?